<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_nuevo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_nombre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_apellidos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellidos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_dni">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DNI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_apellidos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_dni">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guardar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="lista">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>501</width>
     <height>411</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>124</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NOMBRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>APELLIDOS</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DNI</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="input_buscar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Buscar... 3 letras m√≠nimo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>260</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eliminar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_editar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkIcono">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Usar icono en el tray</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
